### DISCUSSION TITLE
Vscode adwaita:dark not working for top bar

### DISCUSSION THREAD

**Discussion by carsonwright** [elementary/triage#552] [Q&A]
The top bar of elementary OS uses adwaita light theme when I set the system theme to dark. Is there a work around. There are a couple apps that behave this way.

**Comment by jeremypw** [comment]
This issue has been raised before - see [#267 ](https://github.com/elementary/triage/issues/267)  and https://github.com/elementary/gala/issues/956.

Looks like it will might be resolved by OS8.

**Comment by jeremypw** [comment]
Just checked - its not fixed yet by OS8 Early Access under either X or Wayland
